config/marklogic-sink-marklogic-ssl.properties

```diff
@@ -59,6 +58,9 @@ ml.dmsdk.threadCount=8
 # Optional - a comma-separated list of collections that each document should be written to
 ml.document.collections=kafka-data
 
+#Optional - set this to false to prevent marklogic from adding topic as collection on documents 
```

Comment:
I think this is a nice feature to have, but it should default to false. The documentation for all the properties for the connector will list it so that a user is aware of it. 

I'd rename it as well to "ml.document.addTopicToCollections", as "ml.document.collections" is a string value and thus I don't think it should have child properties below it. "addTopicToCollections" can then be the name of the attribute in DefaultSinkRecordConverter, as opposed to "addTopicIndicated", which is not consistent with the property name. 

Also, for consistency with the other comments, there should be a space between "#" and "Optional". And "marklogic" should be "MarkLogic" - although it's not MarkLogic doing the adding, it's the connector. I recommend a comment of:

"# Optional - set this to true so that the name of the topic that the connector reads from is added as a collection to each document inserted by the connector"